docs-source/content/samples/simple/domains/model-in-image/_index.md

```diff
@@ -0,0 +1,558 @@
+
+1.  Tag the image for the remote Docker registry, for example:
+
+```
+docker tag model-in-image:v1 my.remote.registry.com/model-in-image:v1
+```
+
+2.  Push the image to the remote Docker registry, for example:
+
+```
+docker push my.remote.registry.com/model-in-image:v1
+```
+
+3. Create the pull secret for the remote Docker registry:
+
+```
+ kubectl -n <domain namespace> create secret docker-registry <secret name> \
+     --docker-server=my.remote.registry.com \
+     --docker-username=your.email@some.com \
+     --docker-password=your-password \
+     --docker-email=your.email@some.com
+
+```
+
+4. Update the domain template file `$SAMPLEDIR/k8s-domain.yaml.template` to provide the `imagePullSecrets`:
+
+```
+  imagePullSecrets:
+  - name: <secret name>
+
+```
+
+This domain template file will be used in the next step ([Create and deploy your Kubernetes resources](#create-and-deploy-your-kubernetes-resources)) when it creates your final domain resource file.
+
+5. Export the environment variables for the image name and tag using the same values in step 1:
+
+```
+export MODEL_IMAGE_NAME="my.remote.registry.com/model-in-image"
+export MODEL_IMAGE_TAG="v1"
+```
+
+These environment variables will be used in the next step ([Create and deploy your Kubernetes resources](#create-and-deploy-your-kubernetes-resources)) when it creates your final domain resource file.
+
+### Create and deploy your Kubernetes resources
+
+To deploy the sample operator domain and its required Kubernetes resources, use the sample script, `$SAMPLEDIR/run_domain.sh`, which will perform the following steps for you:
+
+  - Deletes the domain with a `DomainUID` of `domain1` in the namespace, `sample-domain1-ns`, if it already exists.
+  - Creates a secret containing your WebLogic administrator user name and password.
+  - Creates a secret containing your Model in Image runtime encryption password:
+    - All model-in-image domains must supply a runtime encryption secret with a `password` value.
+    - It is used to encrypt configuration that is passed around internally by the Operator.
+    - The value must be kept private but can be arbitrary: you can optionally supply a different secret value every time you restart the domain.
+  - Creates secrets containing your RCU access URL, credentials, and prefix (these are unused unless the domain type is `JRF`).
+  - Creates a config map containing an additional WDT model properties file, `$SAMPLEDIR/model1.20.properties`.
+  - Generates a domain resource YAML file, `$WORKDIR/k8s-domain.yaml`, using `$SAMPLEDIR/k8s-domain.yaml.template`.
+  - Deploys `k8s-domain.yaml`.
+  - Displays the status of the domain pods.
+
+The script expects `WDT_DOMAIN_TYPE` and `WORKDIR` to already be initialized. (See [Prerequisites for all domain types](#prerequisites-for-all-domain-types).)
+
+Run the script:
+
+  ```
+  $SAMPLEDIR/run_domain.sh
+  ```
```

Comment:
dies -- ```$ p $SAMPLEDIR/run_domain.sh 
@@ Info: Running 'run_domain.sh'.
@@ Info: WORKDIR='/home/mark/operator/mii/work'.
@@ Info: Deleting weblogic domain 'sample-domain1' if it already exists
+ kubectl -n sample-domain1-ns delete domain sample-domain1 --ignore-not-found
@@ Info: Creating weblogic domain secret
Error from server (NotFound): namespaces "sample-domain1-ns" not found
```

Reply:
i had to create the ns first

Reply:
after that it failes with this: 
```@@ Info: Creating domain resource yaml '/home/mark/operator/mii/work/k8s-domain.yaml'.
@@ Info: Running 'create_domain.sh'.
@@ Info: WORKDIR='/home/mark/operator/mii/work'.
@@ Info: Creating domain resource file '/home/mark/operator/mii/work/k8s-domain.yaml' from '/home/mark/operator/mii/weblogic-kubernetes-operator/kubernetes/samples/scripts/create-weblogic-domain/model-in-image/k8s-domain.yaml.template'
sed: -e expression #1, char 36: unknown option to `s'
```

Reply:
@markxnelson  

In theory you should have been following the sample directions, which redirect to 'quick start' directions for setting up the operator and ns.  But I can have run_domain.sh report an error if the ns isn't found, or (more complicated) there's no running operator monitoring the namespace.   Up to you.  But if so, I'd prefer to do this after RC1.

Side note: post RC1 I'd like to move the 'create_domain.sh' script into a separate explicit step in the sample, instead of having it called from 'run_domain.sh'.  Plus call it something like 'stage_domain_resource.sh'.